internal/storage/storage_handle.go

```diff
@@ -91,8 +92,10 @@ func NewStorageHandle(ctx context.Context, clientConfig StorageClientConfig) (sh
 		wrapped:   httpClient.Transport,
 		UserAgent: clientConfig.UserAgent,
 	}
+
+	clientOpts := append(clientConfig.ClientOptions, option.WithHTTPClient(httpClient))
 	var sc *storage.Client
-	sc, err = storage.NewClient(ctx, option.WithHTTPClient(httpClient))
+	sc, err = storage.NewClient(ctx, clientOpts...)
```

Comment:
I think we need to add this parameter in [storage_handle_test ](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/storage/storage_handle_test.go#L39)also

Reply:
Haven't added, since default value will be nil only. 

But I'll add unit for this also in storage_handle_test.go class 
Thanks!